package.json

```diff
@@ -133,6 +133,7 @@
     "nyc": "^15.1.0",
     "pidtree": "^0.5.0",
     "prettier": "^1.19.1",
+    "regenerator-runtime": "^0.13.7",
```

Comment:
Do we really need this devDependency? I'm unsure ...
`npm list regenerator-runtime` shows now:
```
mocha@8.3.0 D:\mocha-fork\mocha
+-- @babel/preset-env@7.12.11
| `-- @babel/plugin-transform-regenerator@7.12.1
|   `-- regenerator-transform@0.14.5
|     `-- @babel/runtime@7.12.5
|       `-- regenerator-runtime@0.13.7 
`-- rewiremock@3.14.3
  `-- babel-runtime@6.26.0
    `-- UNMET DEPENDENCY regenerator-runtime@0.11.1

npm ERR! missing: regenerator-runtime@0.11.1, required by babel-runtime@6.26.0
```
I'm not worrying about the error ... I think.
What do you think of upgrading _@babel/preset-env_ ? There is a minor semver.

Reply:
Good idea. Done. The result seems to be the same.